packages/loader/container-loader/src/container.ts

```diff
@@ -1509,9 +1513,15 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 this._audience.addMember(priorClient.clientId, priorClient.client);
             }
 
+            const deltaManagerForCatchingUp =
+                this.mc.config.getBoolean("Fluid.Container.CatchUpBeforeDeclaringConnected") === true ?
```

Comment:
nit: No need to check "=== true"

Reply:
I prefer to be explicit that it must be set to true (but you're right the code works either way)